Oceanus

takamotobiz opened the pull request:
３タイル化のプルリクエストです。
従来のgeoloniaタイルに、oceanus（北方領土と国境線含む）とgeolonia-water（描画高速化のための水面）を加えています。ソースのURLはわからないので、`mbtiles://`表記にしています。
以下、このスタイルで配信中です。
https://tsgl.internal.geolonia.com/oceanus2/styles/2tile-splitwater/#2/15.62/88.95
添付はスタイルの概要になります。
[Style_20210610.xlsx](https://github.com/geolonia/basic/files/6627404/Style_20210610.xlsx)

takamotobiz committed:
3Tile

keichan34 commented on style.json:
```diff
@@ -3,10 +3,13 @@
+    },
+    "geolonia-water": {
+      "type": "vector",
+      "url": "mbtiles://{water}"
```
あ、すみません、こちらは

```
https://api.geolonia.com/v1/sources_v2?ver=water&key=YOUR-API-KEY
```

でお願いします

keichan34 commented on style.json:
```diff
@@ -3,10 +3,13 @@
       "type": "vector",
-      "maxzoom": 6,
-      "url": "https://cdn.geolonia.com/tiles/natural-earth.json"
+      "url": "mbtiles://{oceanus}"
```
```
https://cdn.geolonia.com/tiles/oceanus.json
```

takamotobiz replied:
ご指摘の2件のURLを修正しました。

keichan34 reviewed (commented):
sources の URL を修正お願いします :bow:

takamotobiz committed:
replace url

keichan34 committed:
tileserverのmetadataに向ける

naogify committed:
Set minzoom:8 for geolonia source

`oceanus`  (https://cdn.geolonia.com/tiles/oceanus.json) が、`max-zoom:7` に設定されていて、`geolonia` が`min-zoom:6` で一部レイヤーが被っていました。

なので、`geolonia` の`min-zoom`を8にして解決しました。

keichan34 commented:
@naogify https://deploy-preview-12--style-basic.netlify.app/#9.71/13.4944/123.5337 ここの境界線が被ってるのも確認してもらってもいいですか？

keichan34 commented:
@naogify あと、ここの神田川なんですが、

https://deploy-preview-12--style-basic.netlify.app/#18.09/35.710642/139.730099

従来のbasicでみると太くなってる(線ではなくてポリゴンなのかな。。？)んだけど、これも見てもらってもいいですか？

miya0001 commented:
川は `water` っていうポリゴンですね。線のデータは `waterway-river` っていう線データで、それぞれ別々に存在しています。

![](https://www.evernote.com/l/ABUCRt5X2vBDNLWHwWxzlSUrrlNQzJggpFIB/image.png)

naogify commented:
レビューチェックリスト
 
- [x] oceanusのレイヤーで`maxzoom:8`以上がないか確認
- [x] geoloniaのレイヤーで`minzoom:8`未満がないか確認
- [x] geoloniaのレイヤーの`water-`がつくレイヤーで、表示がおかしくなっているものがないか確認

naogify committed:
set mazoon:8 for oc-boundary-land-level-1

keichan34 committed:
Merge remote-tracking branch 'origin/master' into oceanus2

naogify committed:
日本のlake と river が非表示になっていたので修正

naogify committed:
Merge branch 'oceanus2' of https://github.com/takamotobiz/basic into oceanus2

naogify committed:
日本のlake と river が非表示になっていたので修正

naogify committed:
Revert "日本のlake と river が非表示になっていたので修正"

This reverts commit 3d0dcb2c8c2b583cc59ab51aada37d1d639bfa39.

naogify committed:
some fix

naogify committed:
move river-lake layer

naogify commented:
@miya0001 @keichan34 

琵琶湖以外のスタイルは不具合なく表示されていることを確認しました。またお手すきの時に見て頂けますと…！（琵琶湖の修正は高本さんに依頼中です）

https://deploy-preview-12--style-basic.netlify.app/

keichan34 commented:
ありがとうございました！

takamotobiz commented:
@keichan34 以下にて確認したところ、択捉島の南西側に緑色のポリゴンが１つ表示されています。zoomlevel=8以上で表示されるので、Tilemakerの要素ではないかと思います。
tsglでは表示されていなかったように思いますが、何かわかりますか？
https://deploy-preview-12--style-basic.netlify.app/#9.07/44.6582/147.0595
![image](https://user-images.githubusercontent.com/74547674/121790958-c8cf1880-cc1f-11eb-960d-e98f62c674f6.png)

takamotobiz commented:
`sourceLayer:park`みたいですね。OSMのラスターで確認したところ、画像のような感じでした。
![image](https://user-images.githubusercontent.com/74547674/121792726-6d0e8a80-cc33-11eb-9a80-a3c338dea757.png)

keichan34 commented:
文字もちょっと途切れてしまっていますね。。なんでだろう

<img width="239" alt="Screen Shot 2021-06-13 at 13 18 25" src="https://user-images.githubusercontent.com/1064513/121795246-f9c44300-cc49-11eb-85c0-1027723624c2.png">

takamotobiz commented:
@keichan34 琵琶湖問題を修正したタイルを、tsglのoceanus2/oceanus.mbtiles におきました。
お手数ですが、以下へのデプロイ方法がわからないので、助けていただけると。
https://deploy-preview-12--style-basic.netlify.app

takamotobiz commented:
@keichan34 
北方領土の件、`"id": "water-river-lake-ja"`にて北方領土の湖が表示されているのが問題の一つだと思います。
確か、ロシアの”湾”は、海なのに属性が”lake”となっていたと記憶しています。
<img width="852" alt="image" src="https://user-images.githubusercontent.com/74547674/121796595-2c276d80-cc55-11eb-9684-d60684a33a67.png">

takamotobiz commented:
@keichan34 北方領土の件、OSM作成のタイルを使用している状態で、スタイル`"id": "water-river-lake-ja"`を追加していただいた影響が出ているようです。また、parkについても同様で、Tilemakerには存在しなかった図形要素がOSMでは作成され問題が発生しているように思います。
OSMで作成した高ズームタイルにて可能な対策（SQLで削除や属性付与）は実績がないので、方針決めが必要なのかなと思います。
![image](https://user-images.githubusercontent.com/74547674/121800369-4b7dc500-cc6c-11eb-956e-9f159b92ece0.png)

naogify committed:
move water-river-lake-ja for fix bug in northen area

naogify commented:
@takamotobiz @keichan34 

> 以下にて確認したところ、択捉島の南西側に緑色のポリゴンが１つ表示されています。zoomlevel=8以上で表示されるので、Tilemakerの要素ではないかと思います。

ご指摘ありがとうございます! この件、`"id": "water-river-lake-ja"` が `park` より上のレイヤーで表示されていたので、 下のレイヤーに移動させる事で修正しました。

![スクリーンショット 2021-06-13 18 18 44](https://user-images.githubusercontent.com/8760841/121801745-d57d5c00-cc73-11eb-8602-263288936e17.png)


https://deploy-preview-12--style-basic.netlify.app/#9.07/44.6582/147.0595

takamotobiz commented:
@naogify 
いろいろ対応いただいて、ありがとうございました！
netlify側はOMTで、tsglはTilemakerなので、ちょっと混乱していました。
park問題を含めOMT版としては、ほぼこれで完成と理解をしています。
TileMaker版へのフィードバックの整理が必要そうですね。

keichan34 commented:
@miya0001 https://deploy-preview-12--style-basic.netlify.app/#11.1/35.1027/135.9447
琵琶湖を修正する oceanus のタイルセットを更新しました(パス更新していないので、キャッシュが効いてるかもしれない)

miya0001 commented:
LGTM

keichan34 commented:
マージしました。皆様ありがとうございました！
